packages/bpk-tokens/src/setOpacity.js

```diff
@@ -19,7 +19,10 @@
 import Color from 'color';
 
 const setOpacity = (colorToken, opacity) => {
-  if (!parseFloat(opacity)) {
+  // Number properties compat is low, so we'll stick to the globals
```

Comment:
I don't really understand what point this is trying to make? Is compatibility with `isNaN` good enough? Or isn't it?

Reply:
`0` is a valid opacity value, but it currently causes an error, because `0 == false`. 

Using `isNaN` is more thorough because it works with `0`.

Reply:
I can see that, but I still don't understand the comment.

Reply:
Ah, sorry I misunderstood your question.

So when I added `isNaN`, eslint complained. I searched for `isNaN` in our codebase to see what we do in other situations, and every other usage of it has this comment, so it seems like a convention, and I copied it over to here too.

Reply:
Maybe the comment would be more useful if it said `isNaN is currently unsupported in IE, but that's fine because... `

Reply:
But then I have to update it everywhere FINE I'll do it 😝

Reply:
Done, try now 😇